src/demo/demo-state.ts

```diff
@@ -99,15 +99,15 @@ export const hentReservasjonKRR = (): boolean => {
 export const settAutentiseringsInfo = () => {
     settILocalStorage(DemoData.AUTENTISERINGS_INFO, JSON.stringify({
         securityLevel: InnloggingsNiva.LEVEL_3,
-        isLoggedIn: true,
```

Comment:
Ikke en høyde jeg dør på... men jeg synes det er bedre å bruke `isLoggedIn` i stedet for `loggedIn`

Reply:
Enig. Det er en (halvdårlig) grunn til endringen. Kan vise deg når du kommer!